### PR TITLE
DOC: copula in user guide and examples

### PR DESCRIPTION
**Pull request by tupui**
Follow up of #7408 which adds Copula.

This PR adds some documentation in the user guide and an example notebook.

cc @josef-pkt

### CODE REVIEW

**Comment by tupui** [comment]
@bashtage it would be great if you could have a look. Might be good to have another pair of eyes on this topic. Thanks 🙏

File: docs/source/distributions.rst
Comment (bashtage) [inline docs/source/distributions.rst:61]: Copulas or perhaps Copulae

**Review by bashtage — CHANGES_REQUESTED** [review]
One small grammar mistake.  I will merge once fixed. Thanks.

File: docs/source/distributions.rst
Comment (tupui) [inline docs/source/distributions.rst:61]:
```suggestion
parameters. Copulae are used to construct a multivariate joint distribution and
```

**Comment by tupui** [comment]
Thank you @bashtage

**Comment by bashtage** [comment]
Thanks.

**Pull request merged**
